iocore/net/UnixNetVConnection.cc

```diff
@@ -270,6 +270,21 @@ read_from_net(NetHandler *nh, UnixNetVConnection *vc, EThread *thread)
     close_UnixNetVConnection(vc, thread);
     return;
   }
+
+  if (vc->read.error) {
```

Comment:
@jacksontj here should be
+  // if it is not enabled and got error from polling
+  if (!s->enabled && s->error) {

we should do read first if we get both of read enabled and error.

e.g.: 
remote do a shutdown() on socket, we will get EVENTIO_READ and EVENTIO_ERROR here.